.gitignore

```diff
@@ -86,7 +86,7 @@ celerybeat-schedule
 *.sage.py
 
 # Environments
-.env
+.env*
```

Comment:
Why this change?

Reply:
Because I have different env files for different environments, that is: .env.s3 and .env.minio